to_data_library/data/transfer.py

```diff
@@ -294,7 +296,7 @@ def s3_to_gs(self, s3_connection_string, s3_bucket_name,
         logs.client.logger.info(f'Found {str(s3_files)} files in S3')
 
         # For every key found in s3, download to local and then upload to desired GS bucket.
-        s3_client = s3.Client(parsed_connection['region'])
+        s3_client = s3.Client(aws_session, parsed_connection['region'])
```

Comment:
please remove all references to parsed connection as its no longer passed to the client